frontend/util.lua

```diff
@@ -1468,4 +1468,31 @@ function util.wrapMethod(target_table, target_field_name, new_func, before_callb
     return wrapped
 end
 
+--- Calculate partial digest of an open file. To the calculating mechanism itself,
```

Comment:
Could you put this below util.getFormattedSize(size) or some similarly appropriate location?